public/index.css

```diff
@@ -3,6 +3,41 @@
   --human-pink: #fa759e;
 }
 
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  /* text-align: center; */
+  color: #164176;
+}
+
 .title {
   color: var(--human-blue);
+  text-align: center;
+}
+
+.steps {
+  display: flex;
+  padding: 5% 10%;
+}
+
+.step {
+  padding: 0 3%;
+  font-size: 16px;
```

Comment:
I think the default `font-size` is `16px` so this line is redundant.

Reply:
corrected.